src/picongpu/include/particles/bremsstrahlung/SynchrotronFunctions.hpp

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright 2015 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/cursor/Cursor.hpp"
+#include "cuSTL/cursor/navigator/PlusNavigator.hpp"
+#include "cuSTL/cursor/tools/LinearInterp1D.hpp"
+#include "cuSTL/cursor/BufferCursor.hpp"
+#include "simulation_defines.hpp"
+#include <boost/shared_ptr.hpp>
+#include <boost/array.hpp>
+#include <boost/numeric/odeint.hpp>
+#include <boost/math/special_functions/bessel.hpp>
```

Comment:
please answer this other question from https://github.com/ComputationalRadiationPhysics/picongpu/pull/1299#discussion_r47224418

**new pre-compiled library** ([docs](http://www.boost.org/doc/libs/1_59_0/libs/math/doc/html/math_toolkit/main_tr1.html#math_toolkit.main_tr1.usage_recomendations))

careful: this is [not a header-only library](https://gist.github.com/ax3l/7450482) and currently not used by others. needs adding in `doc/INSTALL.md` and `CMakeLists.txt` if this shall be used. also needs update in compile suite and various cluster boost libraries.

-> might be possible without compile but pre-compiled versions should be preferred since our compile times do not need to be increased even further (quote: *"speed up compile times, reduce object files sizes (since there are no templates being instantiated any more), and also speed up debugging runtimes [...]"*)

Can you decide on which *pre-compiled* version of math you want to use (math_c99* or math_tr1*)? We will add that precompiled one instead of adding compile time to PIConGPU.